plugins/inputs/rabbitmq/README.md

```diff
@@ -61,6 +61,8 @@ For additional details reference the [RabbitMQ Management HTTP Stats](https://cd
   - messages_ready (int, messages)
   - messages_unacked (int, messages)
   - queues (int, queues)
+  - cluster_nodes (int, member nodes)
+  - running_nodes (int, nodes up)
```

Comment:
All the other metrics are noun_adjective, these new ones are adjective_noun

Reply:
Good point, then I propose
```
cluster_nodes => nodes_member
running_nodes => nodes_running
```

Reply:
How about `clustering_listeners` and `amqp_listeners`?

Reply:
That would be 100% accurate. Agreed, let the user decide the practical meaning of those metrics then.